openshift/3.11/templates/jenkins/generic-pipeline.yaml

```diff
@@ -49,48 +49,48 @@ objects:
       lastVersion: 0
 parameters:
   - name: NAME
-    displayName: Pipeline Name
-    description: Name of the pipeline, e.g cicd, promote-to-test, sonarqube. A -pipeline suffix will be appended to the name provided.
+    displayName: "Pipeline Name"
```

Comment:
Double quotes appear to be required for OCP4.0.  Although it's not entirely consistent behaviour based on my initial experience.